postgres/templates/statefulset.yaml

```diff
@@ -26,6 +26,34 @@ spec:
       imagePullSecrets:
       - name: {{ .Values.imagePullSecrets }}
       {{- end }}
+      {{- if .Values.initData.enabled }}
+      initContainers:
+      - name: init-data
+        image: {{ .Values.initData.image }}
+        env:
+        - name: OS_PROVIDER
+          value: {{ .Values.initData.objectStorage.provider | quote }}
+        - name: OS_BUCKET
+          value: {{ .Values.initData.objectStorage.bucket | quote }}
+        - name: OS_ACCESS_KEY_ID
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.initData.objectStorage.secretKeyRefName }}
+              key: access-key-id
+        - name: OS_SECRET_ACCESS_KEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.initData.objectStorage.secretKeyRefName }}
+              key: secret-access-key
+        - name: OS_HOST
+          value: {{ .Values.initData.objectStorage.host | quote }}
+        command:
+        {{- include "postgres.initData" . | indent 10 }}
```

Comment:
I think it's more clear to have the shell script in the YAML instead of a helm helper.

Reply:
I think the script is not easier to read in the YAML; I would like to move it in a docker image instead.